### ISSUE TITLE
Fetch AST or Code for a method/function

### ISSUE THREAD

**Issue by asgrim** [Roave/BetterReflection#130]
Add new methods to `ReflectionFunctionAbstract` which fetch AST or code representation:
* `ReflectionFunctionAbstract::getAst()` - returns the AST for the code within the function
* `ReflectionFunctionAbstract::getBodyCode()` returns the code of the function body itself (using `PhpParser`'s printer functionality)

**Comment by Ocramius** [comment]
:+1: VERY (***VERY***) useful!

**Comment by asgrim** [comment]
Note the `getBodyCode` would not get the surrounding syntax, it would only get the actual body of the function. See #133 for exporting a whole reflection. For example, given you have a reflection of this method:

```php
private function add($a, $b)
{
    return $a + $b;
}
```

Then calling `getBodyCode()` would result in just:

```php
return $a + $b;
```

**Comment by asgrim** [comment]
Fixed by #135
